tasks/connectors/edgescan/lib/kenna_api.rb

```diff
@@ -11,13 +11,18 @@ def initialize(options)
           @kenna_api_key = options[:kenna_api_key]
           @kenna_connector_id = options[:kenna_connector_id]
           @output_dir = "#{$basedir}/#{options[:output_directory]}"
+          @skip_autoclose = false
+          @max_retries = 3
+          @kdi_version = 1
         end
```

Comment:
Hi) I have reviewed the changes in this connector and all these changes look reasonable. There is one point that I am currently trying to investigate - the possibility of using the KENNA KDI format version 2. So I think it would be better to use version 2

Reply:
Yeah I don't see why not. Tested and updated in 3f92116